src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -164,22 +164,30 @@ A Navitia Line is created to group one or several Navitia Routes when they are c
 
 | NTFS file | NTFS field | Constraint | GTFS file | GTFS field | Note |
 | --- | --- | --- | --- | --- | --- |
-| trips.txt | route_id | Required | trips.txt | route_id | cf. NTFS `route_id` definition above to specify the propert reference. |
+| trips.txt | route_id | Required | trips.txt | route_id | cf. NTFS `route_id` definition above to specify the proper reference. |
 | trips.txt | service_id | Required | trips.txt | service_id |  |
 | trips.txt | trip_id | Required | trips.txt | trip_id |  |
-| trips.txt | trip_headsign | Optionnal | trips.txt |  | `trip_short_name`, of if empty `trip_headsign` |
-| trips.txt | block_id | Optionnal | trips.txt | block_id |  |
-| trips.txt | company_id | Required | routes.txt | agency_id | The company corresponding to the agency_id of the trip's route_id |
-| trips.txt | physical_mode_id | Required |  |  | use the `route_type` See "Mapping of route_type with modes" chapter |
-| trips.txt | trip_property_id | Optionnal | trips.txt |  | (1) |
+| trips.txt | trip_headsign | Optional | trips.txt |  | `trip_short_name`, of if empty `trip_headsign` |
+| trips.txt | block_id | Optional | trips.txt | block_id |  |
+| trips.txt | company_id | Required | routes.txt | agency_id | The company corresponding to the `agency_id` of the trip's `route_id` |
+| trips.txt | physical_mode_id | Required |  |  | use the `route_type` See ["Mapping of route_type with modes"](#mapping-of-route_type-with-modes) chapter |
+| trips.txt | trip_property_id | Optional | trips.txt |  | (1) |
 | trips.txt | dataset_id | Required |  |  | The `dataset_id` provided (cf. [gtfs2ntfs.md](./gtfs2ntfs.md) ) |
-| trips.txt | geometry_id | Optionnal | trips.txt | shape_id |  |
+| trips.txt | geometry_id | Optional | trips.txt | shape_id |  |
 
 (1) The `trip_property` object is a complex type with additional properties :
 + `trip_property_id`: should be generated by the reader.
 + `wheelchair_accessible`: possible values are the same in both GTFS and NTFS
 + `bike_accepted`: corresponding to the GTFS `bikes_allowed` property. Possible values are the same in both GTFS and NTFS.
-Be carefull to only create necessary trip_properties and avoid dupplicates.
+Be careful to only create necessary `trip_properties` and avoid duplicates.
+
+**_"Source" complementary code :_**
+
+A complementary object_code should be added to each network with the following properties :
```

Comment:
```suggestion
A complementary object_code should is added to each vehicle journey with the following properties :
```

Reply:
Done in f312ca02c32f3671a66a420fbe803478de4cc8d1.